app/controllers/shelters_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
 
   def show
     shelter
+    @reviews = @shelter.reviews
```

Comment:
all instance variables in the `show` action methods are available from the `show` view page